config/dev.json

```diff
@@ -16,5 +16,6 @@
     "appId": "saturnnonprod-O9lUP",
     "apiKey": "AQEBBAEkx4iE2KxNyI7Wx08EwU1ycTM7E4FMSmaibbMUQxNU6uQvuAJt7fyABAtFYSYfgEE"
   },
-  "tosUrlRoot": "https://us-central1-broad-workbench-tos-dev.cloudfunctions.net/tos"
+  "tosUrlRoot": "https://us-central1-broad-workbench-tos-dev.cloudfunctions.net/tos",
+  "isJobManager": "false"
```

Comment:
Is this intended to be the feature flag? I don't think the config is the right place for this, since it's not going to vary by environment. What about just exporting a value from a file instead? When we get proper dev/prod environments for JMUI though, those URLs will want to go here.
I think we can come up with a more descriptive name, too. Perhaps something like `linkToJobManager`?
Also, this can be a proper boolean rather than a string.